softwerkskammer/lib/groups/views/get.pug

```diff
@@ -33,9 +33,10 @@ block content
         strong #{t('general.address')}: #{' '}
         a(href='mailto:' + group.id + '@softwerkskammer.org')
           | #{group.id}@softwerkskammer.org
-      p
-        a(href='/mailsender/contactGroupContactPersons/' + group.id)
-          | Ansprechpartner kontaktieren
+        if (userIsGroupMember && group.contactTheOrganizers)
```

Comment:
I'm not really sure and I didn't try it out but isn't there's something important about indentation in pug markup? Because I think the `p` tag no has more indentation (or is this just a consequence of the `if` statement?).

Reply:
I tested this one in the running service and it worked.
But I have not used jade/pug before, not sure if there is a better way to do this.